emr-sql/pom.xml

```diff
@@ -134,6 +134,12 @@
             <artifactId>hadoop-client</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
ditto, move this dependency to `examples` module

Reply:
1